provision/ansible/playbooks/install-sync-gateway-source.yml

```diff
@@ -42,38 +43,37 @@
 # Create sg_accel user
 - hosts: sync_gateway_index_writers
   any_errors_fatal: true
+  become: yes
   tasks:
   - include: tasks/create-sg-accel-user.yml
 
 # Download source and build
 - hosts: sync_gateways
   any_errors_fatal: true
+  become: yes
   vars:
     branch:
     build_flags:
 
   tasks:
-  - name: clone sync gateway git repo
-    git: repo=https://github.com/couchbase/sync_gateway.git
-         dest=/home/centos/sync_gateway
-         recursive=no
-  - name: checkout feature branch
-    shell: git checkout {{ branch }} chdir=/home/centos/sync_gateway
-  - name: update submodules
-    shell: git submodule init && git submodule update chdir=/home/centos/sync_gateway
-
-  # Build with flags if flags are defined, by default build_flags=""
-  - debug: msg="Building with flags {{ build_flags }}"
-  - name: build sync_gateway
-    shell: ./build.sh {{ build_flags }} chdir=/home/centos/sync_gateway
-
+  - name: install repo tool
```

Comment:
@tleyden Is this the best place for this or should it be install as part of the common tools?